c2corg_api/views/document.py

```diff
@@ -7,11 +7,14 @@
     UpdateType, DocumentLocale, ArchiveDocumentLocale, ArchiveDocument,
     ArchiveDocumentGeometry, set_available_cultures)
 from c2corg_api.models.document_history import HistoryMetaData, DocumentVersion
+from c2corg_api.models.user import User
 from c2corg_api.search.sync import sync_search_index
 from c2corg_api.views import to_json_dict
 from c2corg_api.views.validation import check_required_fields, \
     check_duplicate_locales
 
+from __init__ import serialize
```

Comment:
```
from c2corg_api.views import serialize
```
would be more consistent and explicit. By the way you could use the existing import:
```
from c2corg_api.views import to_json_dict
```